src/docs/product/sentry-basics/search/searchable-properties/events.mdx

```diff
@@ -0,0 +1,717 @@
+---
+title: Properties of Events
```

Comment:
```suggestion
title: Event Properties
```

Titling these pages with event, issue, etc. first is shorter & easier to skim/find